domains/_vercel.nazmul.json

```
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "seraprogrammer",
+        "email": "programmernazmul1@gmail.com",
+        "discord": "1083854941287878726"
+    },
+    "record": {
+        "CNAME": "codervai.vercel.app"
+        "TXT": "vc-domain-verify=nazmul.is-a.dev,741b4962f2334f48af82"
```

CNAME should be on main domain
```suggestion
        "TXT": "vc-domain-verify=nazmul.is-a.dev,741b4962f2334f48af82"
```